src/index.js

```diff
@@ -4,224 +4,164 @@
 
```

Comment:
You might as well open the file in an editor since the diff is garbage/useless because the classes are different anyway.